docs/introduction/composition.md

```diff
@@ -21,15 +21,15 @@ their application operators, who may then declare that their application
 requires that kind of resource.
 
 Composition can be used to build a catalogue of kinds and configuration classes
-of infrastructure that fit the needs and opinions of your organisation. As an
+of infrastructure that fit the needs and opinions of your organization. As an
```

Comment:
This change is fine with me, but both are correct :) https://writingexplained.org/organisation-vs-organization-difference

Reply:
@hasheddan interesting, didn't realize the British allowed both forms. :)

Reply:
Guessing Australia does as well cc @negz :)

Reply:
Working as intended. 🇦🇺

Reply:
https://www.nngroup.com/articles/american-vs-british-english-for-web/